tensorflow/compiler/xla/service/gpu/gpu_executable.h

```diff
@@ -44,18 +45,18 @@ namespace gpu {
 
 // GPU-targeting implementation of the XLA Executable interface.
 //
-// Launches the given CUDA kernel via the StreamExecutor.
+// Launches the given GPU kernel via the StreamExecutor.
 //
 // This is an immutable data type after initialization, and thus thread safe.
 class GpuExecutable : public Executable {
  public:
-  // cubin (i.e. the compiled ptx) may be empty, in which case we leave
-  // compilation up to the GPU driver.
+  using GpuVersion = absl::variant<std::pair<int, int>, int>;
```

Comment:
@jerryyin would it be better to move this type alias to `tensorflow/compiler/xla/types.h`, the same data type would be used in XLA compiler frontend as well.